src/replication_test.c

```diff
@@ -266,8 +267,8 @@ send_mgmt_ack(int fd, zvol_op_code_t opcode, void *buf, char *replica_ip,
 			zrepl_status->state = ZVOL_STATUS_HEALTHY;
 			zrepl_status->rebuild_status = ZVOL_REBUILDING_DONE;
 			(*zrepl_status_msg_cnt) = 0;
-			if(*replica_quorum == 0)
-				state_changed_non_quorum=true;
+			if ( replica_quorum_state == 0 )
```

Comment:
why do we need this variable `state_changed_non_quorum`? why don't we directly set `replica_quorum_state` to 1 here?

Reply:
Yes, we can set the value of `replica_quorum_state` directly to 1. But IMO better to update the variable after sending the response successfully.